packages/core/helper-plugin/lib/src/utils/wrapAxiosInstance/index.js

```diff
@@ -0,0 +1,18 @@
+function wrapAxiosInstance(instance) {
+  const wrapper = {};
+  ['request', 'get', 'head', 'delete', 'options', 'post', 'put', 'patch', 'getUri'].forEach(
+    (methodName) => {
+      wrapper[methodName] = (...args) => {
+        console.log(
```

Comment:
I've asked this a couple of times already: can we please wrap this call in a `NODE_ENV==='development'` check for now?